CHANGELOG.md

```diff
@@ -0,0 +1,22 @@
+# Changelog
```

Comment:
That file should be generated only in master branch. PRs shouldn't contain it. 

One of the ways to achieve that is to add something like [that](https://github.com/F-Secure/pytest-rts/blob/master/.github/workflows/pytest-rts.yml#L44) to publishing step in GitHub actions.